### DISCUSSION TITLE
FastAPI / WebSocket / Docker: Handling disconnections and multiple clients

### DISCUSSION THREAD

**Discussion by shyamgovardhan** [fastapi/fastapi#6259] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
https://github.com/shyamgovardhan/fastapi-websocket-container

CONTAINER = fastapi-websocket-container
IMAGE = fastapi-websocket-image
PORT = 8080
CONTAINER_ID = $(shell docker ps -a --format '{{.ID}}' --filter name=$(CONTAINER))

clean:
	docker stop $(CONTAINER_ID)
	docker rm $(CONTAINER_ID)
	rm -rf app/__pycache__

build:
	docker build -t $(IMAGE) .

deploy: build
	docker run -d --name $(CONTAINER) -p $(PORT):$(PORT) $(IMAGE)

test:
	cd app; uvicorn main:app --reload --port $(PORT)
```

### Description

**tl;dr;**
The [Handling disconnections and multiple clients](https://fastapi.tiangolo.com/advanced/websockets/#handling-disconnections-and-multiple-clients) example works when invoked using **uvicorn**; but it does not work when invoked using a local container as per instructions at [FastAPI in Containers - Docker](https://fastapi.tiangolo.com/deployment/docker/#fastapi-in-containers-docker).

**How to reproduce:**
As shown in the [Makefile](https://github.com/shyamgovardhan/fastapi-websocket-container/blob/main/Makefile), 
- make test: works - message log is displayed below the textbox.
- make deploy: does not work - message log is displayed below the textbox.

### Operating System

Linux

### Operating System Details

Debian 11 (bullseye)

### FastAPI Version

0.79.0

### Python Version

Python 3.10.5

### Additional Context

[requirements.txt](https://github.com/shyamgovardhan/fastapi-websocket-container/blob/main/requirements.txt)
fastapi>=0.68.0,<0.69.0
pydantic>=1.8.0,<2.0.0
uvicorn>=0.15.0,<0.16.0

**Comment by shyamgovardhan** [comment]
**Sample Code:** [fastapi-websocket-container](https://github.com/shyamgovardhan/fastapi-websocket-container)

**Comment by shyamgovardhan** [comment, marked as answer]
Please disregard. 
Had to include websockets in requirements.txt
